### PR TITLE
add some config

### LINKED ISSUE #4246: enhance default value in configuration
should enhance default value in configuration, so user would change less items if using in production.

e.g.

```
--rocksdb_block_cache=4096
--rocksdb_db_options={"max_subcompactions":"4","max_background_jobs":"4"}
```

```
# storage
DEFINE_int32(reader_handlers, 32, "Total reader handlers");
DEFINE_int32(num_worker_threads, 32, "Number of workers");

# graph
DEFINE_int32(num_worker_threads, 0, "Number of threads to execute user queries");

  if (FLAGS_num_worker_threads == 0) {
    FLAGS_num_worker_threads = std::thread::hardware_concurrency();
  }
```

```
DEFINE_bool(enable_partitioned_index_filter, false, "True for partitioned index filters");
```

### PR DESCRIPTION
**Pull request by critical27**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [X] enhancement

#### Issue(s) number: 
close #4246

#### Description:
Add some configs into configuration files which are commonly used.

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [X] N/A

Affects:
- [X] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

Note sure if all these newly added configs are in doc.

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

File: conf/nebula-graphd.conf.default [hunk 1]
```diff
@@ -69,8 +69,14 @@
...
 # storage client timeout
 --storage_client_timeout_ms=60000
+# slow query threshold in us
+--slow_query_threshold_us=200000
 # Port to listen on Meta with HTTP protocol, it corresponds to ws_http_port in metad's configuration file
 --ws_meta_http_port=19559
+# Network IO threads number, 0 means physical cpu core num
+--num_netio_threads=0
+# Worker threads number to handle request, 0 means physical cpu core num
+--num_worker_threads=0
```
Comment (CPWstatic) [inline conf/nebula-graphd.conf.default:79]: We have these two config on line 62-65

File: conf/nebula-storaged.conf.default [hunk 2]
```diff
@@ -112,3 +112,21 @@
...
 --rocksdb_block_based_table_options={"block_size":"8192"}
+
+############### misc ####################
+# Whether remove outdated space data
+--auto_remove_invalid_space=true
+# Network IO threads number
+--num_io_threads=16
+# Worker threads number to handle request
+--num_worker_threads=32
+# Maximum subtasks to run concurrently
```
Comment (wenhaocs) [inline conf/nebula-storaged.conf.default:123]: be more precise "... for admin jobs"

**Review by wenhaocs — COMMENTED** [review]
Overall LGTM on storage part. What was the reason for `snapshot_part_rate_limit`, `snapshot_batch_size` ... appearing only in production config not default config before?

**Comment by Sophie-Xie** [comment]
Will we change it in Open Source edition, or just in Enterprise edition?  cc @xiajingchun

File: conf/nebula-graphd.conf.default (hunk 1 above)
Comment (critical27) [reply conf/nebula-graphd.conf.default:79]: 👌

File: conf/nebula-storaged.conf.default (hunk 2 above)
Comment (critical27) [reply conf/nebula-storaged.conf.default:123]: 👌

**Review by HarrisChu — APPROVED** [review]

**Review by wenhaocs — APPROVED** [review]

**Comment by abby-cyber** [comment]
#1895

**Pull request merged**